### DISCUSSION TITLE
WHY SHORTCUTS KEEPS ON CHANGING SH IN .COMMAND FILE TO BASH

### DISCUSSION THREAD

**Discussion by farnking** [farnking/Test-Dragon-Forum-of-TBots#35] [Q&A]
While running CnTestDragon by clicking the desktop icon in Darwin, something keeps on changing the pyshortcuts generated script file from running in sh to bash.   Need ways to override this secret power.

**Comment by Xueyi-Chen-David** [comment]
I run test program of pyshortcut and change bash to sh, nothing will change after execution. Therefore, it is probablity due to the CnTestDragon.py itself.
